.travis.yml

```diff
@@ -1,12 +1,22 @@
 sudo: false
-
 language: node_js
-
+before_install:
+- mkdir ./tests/configuration
+- ls ./tests
+- 'if [ "$TRAVIS_TRUSTED_INTERNAL_MERGE" = "true" ]; then openssl aes-256-cbc -K $encrypted_7e22a7c28f69_key -iv $encrypted_7e22a7c28f69_iv -in config.tar.enc -out ./tests/configuration/config.tar -d && tar xvf tests/configuration/config.tar; fi'
```

Comment:
nit: avoid the temporary file by omitting the `-out` flag to openssl. Use `tar xvf -` to untar from stdin.